app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -30,7 +39,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class NavigationFragment extends Fragment implements OnMapReadyCallback, OnCameraIdleListener, Subject {
+public class NavigationFragment extends Fragment implements OnMapReadyCallback, OnCameraIdleListener, Subject, SearchView.OnQueryTextListener, SearchView.OnCloseListener {
 
     private final float CAMPUS_DEFAULT_ZOOM_LEVEL = 16f;
     Campus currentCampus = Campus.SGW;
```

Comment:
@adriannadiaz could you resolve this? 🤓 The line it's referencing didn't require changes